extensions/others.go

```diff
@@ -0,0 +1,8 @@
+//go:build ext_test
+
+package extensions
+
+import (
+	_ "github.com/kwilteam/kwil-db/extensions/listeners/spammer"
+	_ "github.com/kwilteam/kwil-db/extensions/resolutions/spam"
```

Comment:
This spam test has obvious value as we've identified critical bugs and performance issues, but it hurts a little to have these so readily enabled in the product code of our main module.

How crazy would it be to instead have a "spam node" build defined in the `test` module, in the vein of https://github.com/kwilteam/kwil-db/pull/570. Maybe my worries about this are somewhat illogical?  If others think it's fine to bake these spam extensions into kwild main module code behind a build tag, I guess I'm alright with it too.  Any opinions @Yaiba @brennanjl ?

Reply:
Seems both are fine, I kind prefer it to be in `test` module if it's not too much work. It seems also more reusable for testing?

Reply:
Yup, I like it in `test` module too. Will update it

Reply:
Doing here: https://github.com/jchappelow/kwil-db/tree/spamd

Making a different docker image so it can't get accidentally built as `kwild:latest`